app/views/submissions/steps/trial/_step02.html.haml

```diff
@@ -9,9 +9,9 @@
 
 .form-group
   = content.label :trait_descriptor_list
-  -#- options = options_for_submission_select(@content.existing_trait_descriptors, :descriptor_name)
-  -#- options.concat options_for_submission_select(@content.new_trait_descriptors, :descriptor_name, id: false)
-  = content.select :trait_descriptor_list, {}, {},
+  - options = options_for_submission_select(@content.existing_trait_descriptors, :trait_name)
+  - options.concat options_for_submission_select(@content.new_trait_descriptors, :trait, id: false)
+  = content.select :trait_descriptor_list, options.html_safe, {},
```

Comment:
@kammerer Can you test this version? I added both an old and a new TD, moved one step further, moved one step back (both were displayed), removed the old TD (the grey one), moved one step further, moved one step back, the new TD was still displayed.